interface.py

```diff
@@ -15,6 +30,32 @@ def build_nn_input(car_list, k):
 		nn_input.append(np.array(param_list).flatten())
 	return np.array(nn_input)
 
-def build_nn_output(normal_list, std_x=1, std_y=1):
-	return np.array(([np.random.normal(mean_x, abs(std_x)) for (mean_x, _) in normal_list],
-		[np.random.normal(mean_y, abs(std_y)) for (_, mean_y) in normal_list]))
+def bound(min_val, max_val, val):
+	"""Bounds a value to be between the range [min_val, max_val]"""
+	if val < min_val:
+		return min_val
+	elif val > max_val:
+		return max_val
+	return val
+
+def build_nn_output(normal_list, max_accel, std_x=1, std_y=1):
+	"""Returns controls corresponding to neural net output means (using normal distribution).
+
+	Args:
+		normal_list: numpy array containing [mean_x, mean_y] 2d np arrays specifying mean of control values.
+		max_accel: all controls should be between -max_accel and max_accel.
+   	std_x: standard deviation for the x control.
+   	std_y: standard deviation for the y control.
+
+  Returns:
+  	(lx, ly): lx is a numpy array of controls in x direction, ly is a numpy array of controls
+  		in the y direction. Each control is just a float.
+	"""
+	assert(type(normal_list) == np.ndarray)
+	assert(type(normal_list[0]) == np.ndarray)
+	assert(len(normal_list[0]) == 2)
+	controls_x = [bound(-max_accel, max_accel, np.random.normal(mean_x, abs(std_x))) 
+		for (mean_x, _) in normal_list]
+	controls_y = [bound(-max_accel, max_accel, np.random.normal(mean_y, abs(std_y)))
+		for (_, mean_y) in normal_list]
```

Comment:
Shouldn't the max acceleration be dependent on both the x and y axis of acceleration? I think the way to do this is to figure out the acceleration magnitude, and if it is greater than max_accel, then scale down the acceleration components by max_accel / magnitude.

Alternatively, we can change the action to be acceleration magnitude + direction. Then the acceleration clipping is a bit easier.